metadata-ingestion/src/datahub/ingestion/source/sql/clickhouse.py

```diff
@@ -145,7 +145,9 @@ class ClickHouseConfig(
     )
     include_materialized_views: Optional[bool] = Field(default=True, description="")
 
-    def get_sql_alchemy_url(self, current_db=None):
+    def get_sql_alchemy_url(
+        self, uri_opts: dict[str, Any] | None = None, current_db: str | None = None
```

Comment:
type annotations need fixes e.g.
```suggestion
        self, uri_opts: Optional[Dict[str, Any]] = None, current_db: Optional[str] = None
```